cmd/dep/init.go

```diff
@@ -152,7 +163,20 @@ func (cmd *initCommand) Run(ctx *dep.Ctx, args []string) error {
 	l = dep.LockFromSolution(soln)
 
 	rootAnalyzer.FinalizeRootManifestAndLock(m, l)
-	gs.FinalizeRootManifestAndLock(m, l)
+	if cmd.gopath {
+		gs.FinalizeRootManifestAndLock(m, l)
+	} else {
+		// Pick the direct dependencies from the above solution and add to manifest
```

Comment:
@carolynvs I just figured out why you asked this question. Went through the epic #186 and came across
>ignore all the existing local information - GOPATH, existing tools, everything - and just have the solver pick out the latest versions it can.

😅 which is totally different from what's being done here. Lemme change that.